templates/base.html

```diff
@@ -84,7 +84,7 @@
                             <li><a href="{% url 'addWorkshop' %}">Zgłoś warsztaty!</a></li>
                             {% if has_workshops %}
                                 <li><a href="{% url 'yourWorkshops' %}">Twoje warsztaty</a></li>
-                            {% endif %}
+                 5           {% endif %}
```

Comment:
Remove this number and fix the indentation.

Reply:
oops